common/fuse.h

```diff
@@ -296,6 +296,11 @@ class Fusor : public Looper {
     return sqrtf(gyro_slope_.z * gyro_slope_.z + gyro_slope_.y * gyro_slope_.y) * (M_PI / 180);
   }
 
+  // Acceleration into twist (one direction) in degrees per second per second
+  float twist_accel() {
+    return fabs(gyro_slope().x);
```

Comment:
no fabs here
If you need to take the absolute value, we can maybe do that in the style, but not here.